distribution/src/main/resources/config/elasticsearch.yml

```diff
@@ -7,8 +7,7 @@
 # The primary way of configuring a node is via this file. This template lists
 # the most important settings you may want to configure for a production cluster.
 #
-# Please see the documentation for further information on configuration options:
-# <https://www.elastic.co/guide/en/elasticsearch/reference/current/settings.html>
+# Please consult the documentation for further information on configuration options.
```

Comment:
We could try https://www.elastic.co/guide/en/elasticsearch/reference/index.html? What do you think?